qiita_db/test/test_software.py

```diff
@@ -872,7 +872,7 @@ def test_eq(self):
         self.assertFalse(a == b)
         # Test difference due to command
         b = qdb.software.Parameters.from_default_params(
-            qdb.software.Command(2).default_parameter_sets.next(),
+            qdb.software.Command(2).default_parameter_sets.__next__(),
```

Comment:
I think the pythonic way to say this, instead of calling \_\_next\_\_()is to say:
next(qdb.software.Command(2).default_parameter_sets)